pkgs/by-name/ki/kickstart/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, nix-update-script
+, testers
+, kickstart
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kickstart";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "Keats";
+    repo = "kickstart";
+    rev = "v${version}";
+    hash = "sha256-GIBSHPIUq+skTx5k+94/K1FJ30BCboWPA6GadgXwp+I=";
+  };
+
+  cargoHash = "sha256-cOcldEte7zxyxzvj7v7uCczs5AQ+v4mMfqmTK9hrv1o=";
+
+  checkFlags = [
+    # remote access
+    "--skip=generation::tests::can_generate_from_remote_repo_with_subdir"
+    "--skip=generation::tests::can_generate_from_remote_repo"
+  ];
+
+  passthru = {
+    updateScript = nix-update-script { };
+    tests.version = testers.testVersion {
+      inherit version;
+      package = kickstart;
+      command = "kickstart --version";
```

This function already sources the version to test for from the package. Also `<mainProgram> --version` is the default command

https://github.com/NixOS/nixpkgs/blob/366b54b81097701b05276d1b9b63622c558abe17/pkgs/build-support/testers/default.nix#L59-L61

```suggestion
      package = kickstart;
```